src/shipit_pulse_listener/shipit_pulse_listener/hook.py

```diff
@@ -93,6 +93,10 @@ def build_consumer(self, pulse_user, pulse_password):
         env = self.parse(body)
         if env is None:
             logger.warn('Skipping message, no task created', hook=self.hook_id)
+        elif isinstance(env, list):
```

Comment:
*Nit*: we could make env a list by default (and call it envs), to reduce the amount of code here.

Reply:
Done !